MetaMask/metamask-extension
# feat: QR-based add NGRAVE ZERO Hardware

## **Description**
Hello,

This PR changes the UI and text to add the NGRAVE ZERO hardware wallet.

Our upcoming release planned for the beginning of July will add QR code compatibility with MetaMask following EIP-4527.



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25080?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/536

## **Manual testing steps**

1. On the hardware wallet, open the 'sync' screen and choose MetaMask
2. Add a QR based wallet on MetaMask (notice that NGRAVE is not 'coming soon' anymore, but listed as a normal option)
3. Scan the QR code from the hardware wallet.

## **Screenshots/Recordings**





### **Before**
![image](https://github.com/MetaMask/metamask-extension/assets/26218337/d67c6182-dd8a-4d57-87df-c2b0ad4a344f)




### **After**

![image](https://github.com/MetaMask/metamask-extension/assets/26218337/088862de-0b21-448e-ba5a-c31610a9d14f)




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
